### DISCUSSION TITLE
could I by pass access service libresbc by by IP not domain and also with digest username only without password

### DISCUSSION THREAD

**Discussion by rahman77889** [hnimminh/libresbc#63] [Q&A]
Hello Everyone,

Our partner using VOS3000, then can't connect to libreSBC

they said don't support using domain and don't support auth passwords just can fill IP and username only

is there any way to bypass access service libreSBC by IP, not domain, and also with digest username only without password ?

because we can't fill domain policy with IP and also can't fill directory/user with empty secret/password

Please help me
Thank you so much

**Comment by hnimminh** [comment]
Hi @rahman77889, 
I my view, there 2 ways to connect to VOS3000

1. Use Inbound/Outbound Interconnection
2. There is no IP authentication function  in access layer, If you need it, you can file a Feature Request for it (* _be responsible to reply all question template_). I'm not promising it will be implemented soon, but I will..

**Reply by rahman77889** [reply]
> Hi @rahman77889, I my view, there 2 ways to connect to VOS3000
> 
> 1. Use Inbound/Outbound Interconnection
> 2. There is no IP authentication function in access layer, If you need it, you can file a Feature Request for it (* _be responsible to reply all question template_). I'm not promising it will be implemented soon, but I will..

Hi @hnimminh 

Refer to the first option, how to use inbound/outbound interconnection without an access layer?
is that any config to bypass the access layer to inbound interconnection, or just fill empty rules on the access layer?

Thank you

**Reply by hnimminh** [reply]
this one for you https://github.com/hnimminh/libresbc/wiki/Interconnection

**Reply by rahman77889** [reply]
> this one for you https://github.com/hnimminh/libresbc/wiki/Interconnection

finnaly I change to domain then it still show 403 forbidden then show 407 many times

then I create access directory user and fill secret, then setup to user agent
but it still show 403 forbidden then show 407 many times

FYI, I've been register IP on whitelist module firewall and also on whitelist on module access service

and also I've been using call flow inbound with IP, but not working too

until now still show still 403 forbidden then show 407 many times

here are the screenshot sngrep, if want pcap please ask me 

![image](https://user-images.githubusercontent.com/4687647/183455434-aa3aff9f-a3b8-4b2d-b4f5-dfe9359bc3bc.png)


I really need this kind project work'swell

Thank You

**Reply by hnimminh** [reply]
@rahman77889 Without enough information we can't not help.
Adding the description, configuration, expected result, actual result will be good idea.
